opta/core/validator.py

```diff
@@ -78,25 +78,35 @@ def validate(self, value: Any) -> List[str]:
         else:
             schema_path = path.join(schema_dir_path, "service.yaml")
 
-        return _get_yamale_errors(value, schema_path, self.module_validator)
+        return _get_yamale_errors(value, schema_path, self.extra_validators)
+
+
+class AwsId(Validator):
+    tag = "aws_id"
+
+    def _is_valid(self, value: Any) -> bool:
+        str_value = str(value)
+
+        return str_value.isdigit() and len(str_value) == 12
```

Comment:
Let's remove the 12 length constraint? No need to add unnecessary validation.

Reply:
AWS defines account ids as having EXACTLY 12 digits

Reply:
That can change down the line... or there can be weird exceptions where this doesn't apply. No big advantage of having this constraint...

Reply:
done